app/global/reducers/intervention/sagas/createSession.js

```diff
@@ -13,13 +13,17 @@ export function* createSession({ payload: { id, lastPosition, type } }) {
   const requestURL = `v1/interventions/${id}/sessions`;
   const sessionNamePrefix =
     type === SessionTypes.CAT_SESSION ? CAT_SESSION_NAME_PREFIX : '';
+  const sessionName =
+    type === SessionTypes.SMS_SESSION
+      ? 'New SMS Campaign'
+      : `${sessionNamePrefix}New Session`;
```

Comment:
Oh, there is a bug, it should be an i18n message here. Remember that the prefix would by at the end in RTL languages. You would need to modify generated `ar.json` so the prefix is at the end. Please fix it using `formatInterventionLanguageMessage` from `app/utils/intlOutsideReact.js` or create a bug ticket.

Reply:
I will create bug ticket, as in createIntervention saga there is the same bug present. I think one should check all creation sagas in CIAS to find all possible places, where such bug can be present.

Reply:
great!